ovirt/resource_ovirt_vm.go

```diff
@@ -457,9 +450,74 @@ func resourceOvirtVMCreate(d *schema.ResourceData, meta interface{}) error {
 func resourceOvirtVMUpdate(d *schema.ResourceData, meta interface{}) error {
 	conn := meta.(*ovirtsdk4.Connection)
 	vmService := conn.SystemService().VmsService().VmService(d.Id())
-	paramVM := ovirtsdk4.NewVmBuilder()
+	vmBuilder := ovirtsdk4.NewVmBuilder()
 	attributeUpdated := false
 
+	// Start update VM Basic parameters (Name, Memory, Cluster, CPU params)
+        if name, ok := d.GetOk("name"); ok {
+                // Define new memory
```

Comment:
Fix the comment or just drop it, because its readable enough.

Reply:
Removed commit - https://github.com/oVirt/terraform-provider-ovirt/pull/198/commits/7fa45520ca4c370cc765d9bd9a0883fea7e409ce